node/router/messages/src/helpers/codec.rs

```diff
@@ -59,8 +59,17 @@ impl<N: Network> Encoder<Message<N>> for MessageCodec<N> {
             .map_err(|_| std::io::Error::new(std::io::ErrorKind::InvalidData, "serialization error"))?;
 
         let serialized_message = dst.split_to(dst.len()).freeze();
-
-        self.codec.encode(serialized_message, dst)
+        #[cfg(feature = "metrics")]
+        let num_bytes = serialized_message.len() as f64;
+        self.codec.encode(serialized_message, dst)?;
+        #[cfg(feature = "metrics")]
+        metrics::histogram_label(
+            metrics::tcp::TCP_ROUTER_MESSAGES_OUTBOUND,
```

Comment:
@ljedrz I noticed in testing with an only validator devnet that this and the router messages inbound metric dont record any values-- is this to be expected?